i18n/translations/en.json

```
@@ -413,16 +413,15 @@
 
 	"license": "License",
 	"license.activate": "Activate it now",
+	"license.activate.label": "Please activate your site",
```

It's not really the site that gets activated but the license (also makes the status text in the stats card easier to understand/mentally connect):

```suggestion
	"license.activate.label": "Please activate your license",
```